grass7/raster/r.in.nasadem/r.in.nasadem.py

```diff
@@ -339,7 +339,9 @@ def createTMPlocation(epsg=4326):
 
     # switch to temp location
     os.environ['GISRC'] = str(SRCGISRC)
-    if grass.parse_command('g.proj', flags='g')['epsg'] != str(epsg):
+    srid = grass.parse_command('g.proj', flags='g')['srid']
```

Comment:
One could also check which version of `g.proj` is used to make sure that also the old version will still work. Something like:
```suggestion
        proj = grass.parse_command('g.proj', flags='g')
        if 'epsg' in proj:
            currepsg = proj['epsg']
        else:
            currepsg = proj['srid'].split('EPSG:')[1]

```